client/login/wp-login/index.jsx

```diff
@@ -36,7 +36,7 @@ import RequestLoginEmailForm from '../magic-login/request-login-email-form';
 import { recordTracksEvent } from 'state/analytics/actions';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 
-class Login extends React.Component {
+export class Login extends React.Component {
```

Comment:
I guess I'm supposed to just export the class, so I can load it in testing? Does this have side effects I'm not aware of?